src/main/java/activeedge/parser/Parser.java

```diff
@@ -223,6 +228,20 @@ public void handleInput(String input) {
                             "c/[CALORIES_BURNT_PER_MINUTE] d/[DURATION_IN_MINUTES]'.");
                     System.out.println("For example, 'add e/Running c/10 d/30'. Enter 'help' for more information.");
                 }
+            } else if (inputSplit[0].equalsIgnoreCase("change")){
+                if (inputSplit.length == 1) {
+                    System.out.println("Please specify what you want to change:");
+                    System.out.println("1. 'change h' to change your height");
+                    System.out.println("2. 'change w' to change your weight");
+                } else if (inputSplit[1].equalsIgnoreCase("h")) {
+                    ChangeHeightCommand.execute();
+                } else if (inputSplit[1].equalsIgnoreCase("w")) {
+                    ChangeWeightCommand.execute();
+                } else {
+                    System.out.println("These are the only change commands: ");
+                    System.out.println("1. change h");

```

Comment:
Shall we change this to the same as in line 234?
"1. 'change h' to change your height"